apps/teams-test-app/src/components/AppInitialization.tsx

```diff
@@ -27,12 +27,10 @@ const NotifySuccess = (): React.ReactElement =>
     title: 'appInitialization.success',
     onClick: {
       withPromise: async () => {
-        app.notifySuccess();
-        return 'called';
+        return JSON.stringify(await app.notifySuccess());
       },
       withCallback: (setResult) => {
-        appInitialization.notifySuccess();
```

Comment:
Do we need to add or update notifySuccess e2e tests?

Reply:
I do think we need to do it as the test app of course changed to display these new values. When I noticed the E2E tasks didn't failed, I stop thinking about it as maybe the current E2E tests do not test the specific content of the `notifySuccess` text box but only that it doesn't error out. I'll check the E2E tests tomorrow and let you know lol

Reply:
What was the answer to this?

Reply:
So, for now I decided not to add nor update the current E2E tests. The reasoning behind this is that, although the explicit response of the function changed, the expected client behavior for using this function is exactly the same which is stopping the loading indicator for the app. As such, the unit tests are properly modified and enhanced to consider when should or not should the function receive a response. But as E2E tests are more of a "black box" testing to see if the behavior is preserved, I wouldn't consider modifying the existing ones 😄